### PR TITLE
updated python issue tracker and pep links

### LINKED ISSUE #2317: Update footer links
A couple of the links in the footer need updating:

# PEP Index

http://python.org/dev/peps/ -> https://peps.python.org

# Issue Tracker

https://bugs.python.org/ -> https://devguide.python.org/triage/issue-tracker/ (or https://github.com/python/cpython/issues)

### PR DESCRIPTION
**Pull request by varunhc**
PR for "Update footer links #2317"

resolves #2317

### CODE REVIEW

File: templates/base.html
Comment (hugovk) [inline templates/base.html]: Thanks! Little suggestion:

```suggestion
          href="https://peps.python.org/peps.rss">
```

**Comment by varunhc** [comment]
@hugovk Thanks! Committed the suggestion.

**Review by hugovk — APPROVED** [review]
Thank you!

**Comment by varunhc** [comment]
@hugovk Is there anything else that needs to be done from my end to get the approval for the PR and get it merged?

**Comment by hugovk** [comment]
No, looks good to me. We now wait for someone with merge rights to check it and trigger the CI.

**Comment by varunhc** [comment]
Oh, got it. Thanks!

**Comment by Jamim** [comment]
Hello @ewdurbin,

While you are here, would you mind reviewing&merging this PR?

Thanks in advance!

**Review by ewdurbin — APPROVED** [review]

**Pull request merged**
